torchtext/experimental/utils/samplers/seqlen_sampler.py

```diff
@@ -0,0 +1,73 @@
+from typing import List
+
+import numpy as np
+from torch.utils.data import Dataset, IterableDataset
+from torch.utils.data.sampler import RandomSampler, Sampler, SequentialSampler
+
+
+class BucketByLengthBatchSampler(Sampler):
```

Comment:
A shorter name?

Reply:
change it to `BucketBatchSampler`, let me know if you have other preferences